src/xdocs/config_coding.xml

```diff
@@ -2180,6 +2207,16 @@ try {
   &lt;property name=&quot;ignoreOverriddenMethods&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Example:</p>
+        <source>
+public class Test{
```

Comment:
missing space before opening brace `{` for the same style as  the other examples.

Reply:
I will change the codestyle and pay attention to it in the future.

Reply:
Done